samples/subsys/fs/littlefs_blk/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
There is already a littlefs sample, samples/subsys/fs/littlefs, why can not it be extended?

Reply:
:arrow_up:

Reply:
good point

Reply:
not addressed

Reply:
@lmajewski please take a look at this comment here

Reply:
@carlescufi @jfischer-no - Maybe this "review view" will help with the discussion...

The reason for a separate test is to address block device specific problems (to test if LBAs are properly written). To do this properly - the pattern write has been used.

Please correct me if I'm wrong, but IMHO it would be better to have two tests than one large? One can always call two tests if required - i.e. `littlefs` and `littlefsblk`.